observation_tickets/Observation_ticket_creator.pyw

```diff
@@ -52,6 +56,103 @@ def quit_func():
     savetxt()
     master.quit()
 
+def check_toi():
+    '''
+    Description
+    -----------
+    Checks to see if a target has been selected, and if so,
+    displays it on the widget
+
+    '''
+    global info_directory
+    link = 'https://docs.google.com/spreadsheets/d/17tqqRnaXuI9q_w1uz35wttQQFGpOIP3hhVXZ5-bYnQA/export?format=csv&id=17tqqRnaXuI9q_w1uz35wttQQFGpOIP3hhVXZ5-bYnQA'
```

Comment:
ideally this should be a config parameter defined in config_reader.py, but if you want to keep the ticket creator more as a 'separate' program from the rest of the code, then I suppose I understand.  Still, maybe you could have a pseudo 'ticket creator config' json file in the same observation_ticket directory, that you just read in as a dictionary with json.load or something to have all of these URLs and passwords within